pypck/connection.py

```diff
@@ -425,9 +426,9 @@ def get_module_conn(
             addr = LcnAddr(self.local_seg_id, addr.addr_id, addr.is_group)
         address_conn = self.address_conns.get(addr, None)
         if address_conn is None:
-            address_conn = ModuleConnection(self, addr)
+            address_conn = ModuleConnection(self.registry, self, addr)
```

Comment:
Wouldn't it be sufficient to just pass the reference to PchkConnectionManager? Do we really need to pass the task registry as an additional parameter?
Any best practice argument would convince me.

Reply:
Yes it is sufficient. I did initially do so but ran into mypy troubles in request_handler. Somehow mypy failed to properly resolve the type of `addr_conn.conn.registry`.

I think I could as alternative not pass it in and instead have a property 
```py
@property
def registry(self) -> TaskRegistry:
    return self.conn.registry
```

I'll try if mypy is ok with that.